src/index.js

```diff
@@ -125,12 +131,16 @@ class Dropzone extends React.Component {
     }
   }
 
-  allFilesAccepted(files) {
-    return files.every(file => accepts(file, this.props.accept));
+  fileAccepted(file) {
+    return accepts(file, this.props.accept);
   }
 
-  allFilesMatchSize(files) {
-    return files.every(file => (file.size <= this.props.maxSize && file.size >= this.props.minSize));
+  fileMatchSize(file) {
+    return file.size <= this.props.maxSize && file.size >= this.props.minSize;
+  }
+
+  allFilesAccepted(files) {
+    return files.every(this.fileAccepted);
```

Comment:
The `fileAccepted` method is bound to the class and the `props` also. This works as expected it's also covered through the test.